lib/adf_variable_defaults.yaml

```
@@ -65,10 +65,14 @@
 #+++++++++++++
 # Available ADF Default Plot Types
 #+++++++++++++
-
 default_ptypes: ["Tables","LatLon","LatLon_Vector","Zonal","Meridional",
                   "NHPolar","SHPolar","Special"]
 
+#+++++++++++++
+# Constants
+#+++++++++++++
+Rgas: 287.04 #[J/K/Kg]=8.314/0.028965
```

Might be good to add a comment here explicitly stating what this is, e.g.:

```suggestion
#Dry Air Gas Constant:
Rgas: 287.04 #[J/K/Kg]=8.314/0.028965
```